builders/integrations/indexers/subsquid.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Index Moonbeam Data with Subsquid
-description: Learn how to use Subsquid, a query node framework for Substrate-based chains, to index and process Substrate and EVM data for Moonbeam and Moonriver.
+title: Index Moonbeam Data with SQD (formerly Subsquid)
```

Comment:
Because `| Moonbeam Docs` gets automatically appended to the titles, this would make the title `Index Moonbeam Data with SQD (formerly Subsquid) | Moonbeam Docs`, which is 64 characters. Needs to be less than 60 characters so maybe just drop "Moonbeam" from the title here as that'll get appended at the end